packages/squiggle-lang/__tests__/ReducerInterface/ReducerInterface_Distribution_test.res

```diff
@@ -119,27 +119,33 @@ describe("eval on distribution functions", () => {
 
 describe("parse on distribution functions", () => {
   describe("power", () => {
-    testParse("normal(5,2) ^ normal(5,1)", "Ok((:pow (:normal 5 2) (:normal 5 1)))")
-    testParse("3 ^ normal(5,1)", "Ok((:pow 3 (:normal 5 1)))")
-    testParse("normal(5,2) ^ 3", "Ok((:pow (:normal 5 2) 3))")
+    testParse("normal(5,2) ^ normal(5,1)", "Ok((:$$block (:pow (:normal 5 2) (:normal 5 1))))")
+    testParse("3 ^ normal(5,1)", "Ok((:$$block (:pow 3 (:normal 5 1))))")
+    testParse("normal(5,2) ^ 3", "Ok((:$$block (:pow (:normal 5 2) 3)))")
   })
   describe("subtraction", () => {
-    testParse("10 - normal(5,1)", "Ok((:subtract 10 (:normal 5 1)))")
-    testParse("normal(5,1) - 10", "Ok((:subtract (:normal 5 1) 10))")
+    testParse("10 - normal(5,1)", "Ok((:$$block (:subtract 10 (:normal 5 1))))")
+    testParse("normal(5,1) - 10", "Ok((:$$block (:subtract (:normal 5 1) 10)))")
   })
   describe("pointwise arithmetic expressions", () => {
     testParse(~skip=true, "normal(5,2) .+ normal(5,1)", "Ok((:dotAdd (:normal 5 2) (:normal 5 1)))")
     testParse(
       ~skip=true,
       "normal(5,2) .- normal(5,1)",
-      "Ok((:dotSubtract (:normal 5 2) (:normal 5 1)))",
+      "Ok((:$$block (:dotPow (:normal 5 2) (:normal 5 1))))",
```

Comment:
dotPow? How did this change from dotSubtract? I imagine this might be outside this PR though.

Reply:
This is really weird. Who is converting the operator names to function names?